src/View/Helper/FormHelper.php

```diff
@@ -135,6 +136,31 @@ public function create($model = null, array $options = [])
     }
 
     /**
+     * Creates a submit button element.
+     *
+     * Overrides parent method to add CSS class `btn`, to the element.
+     *
+     * @param string $caption The label appearing on the button OR if string contains :// or the
+     *  extension .jpg, .jpe, .jpeg, .gif, .png use an image if the extension
+     *  exists, AND the first character is /, image is relative to webroot,
+     *  OR if the first character is not /, image is relative to webroot/img.
+     * @param array $options Array of options. See above.
+     * @return string A HTML submit button
+     * @link http://book.cakephp.org/3.0/en/views/helpers/form.html#creating-buttons-and-submit-elements
+     */
+    public function submit($caption = null, array $options = [])
+    {
+        $class = ['btn'];
+        if (array_key_exists('class', $options)) {
+            $optionsClass = explode(' ', $options['class']);
```

Comment:
You are assuming `class` will be string, it can be array too. So this whole is not necessary imo, if an array is passed 'btn' will be merged and if someone wants to override then can use string.